index.js

```diff
@@ -177,6 +177,7 @@ function setConfig(configObject) {
     clientId: configObject && configObject.clientId || process.env.IOPIPE_TOKEN || process.env.IOPIPE_CLIENTID || '',
     debug: configObject && configObject.debug || process.env.IOPIPE_DEBUG || false,
     network_timeout: 5000,
+    timeout_millis: 50
```

Comment:
any suggestions for a better name? I'm thinking this is not descriptive enough for what it does?